src/components/editor/Bindings/Editor.tsx

```diff
@@ -10,16 +24,129 @@ interface Props {
 }
 
 function BindingsEditor({ loading, skeleton, readOnly = false }: Props) {
+    const theme = useTheme();
+    const jsonTheme =
+        theme.palette.mode === 'dark' ? 'bright' : 'bright:inverted';
```

Comment:
Not a big deal... we probably want to end up moving this into the theme and make it a function that returns the proper setting like how we handle other dark/light mode stuff.